full-service/src/db/account.rs

```diff
@@ -63,9 +85,39 @@ pub trait AccountModel {
         conn: &PooledConnection<ConnectionManager<SqliteConnection>>,
     ) -> Result<(AccountID, String), WalletDbError>;
 
+    /// Create an account.
+    ///
+    /// Returns:
+    /// * (account_id, main_subaddress_b58)
+    #[allow(clippy::too_many_arguments)]
+    fn create(
+        entropy: &[u8],
+        key_derivation_version: u8,
+        account_key: &AccountKey,
+        first_block_index: Option<u64>,
+        import_block_index: Option<u64>,
+        next_subaddress_index: Option<u64>,
+        name: &str,
+        conn: &PooledConnection<ConnectionManager<SqliteConnection>>,
+    ) -> Result<(AccountID, String), WalletDbError>;
```

Comment:
Any reason this needs to be exposed on the `AccountModel` trait, rather than the implementation(s)? Or, alternatively, use this as `_create` and provide default implementations for `create_from_mnemonic` and `create_from_entropy`?